e2e/utils/global-setup.ts

```diff
@@ -19,10 +21,10 @@ export default async function (globalConfig: Config.ConfigGlobals) {
     storage: storageLocation,
   });
 
-  if (
-    process.env.NX_E2E_SKIP_BUILD_CLEANUP !== 'true' ||
-    !existsSync('./build')
-  ) {
+  if (process.env.NX_E2E_SKIP_CLEANUP !== 'true' || !existsSync('./build')) {
```

Comment:
The env variable in the .github files was renamed, but this one isn't. Should it be?

Reply:
They are all renamed.